B-教学案例与实践/B13-AI对联生成案例/src/pai_config/hdfshelper.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
Add license for every code file.

# Copyright (c) Microsoft. All rights reserved.
# Licensed under the MIT license. See LICENSE file in the project root for full license information.

Reply:
added